driver/pparse.mli

```diff
@@ -13,7 +13,11 @@
 (*                                                                        *)
 (**************************************************************************)
 
-(** Driver for the parser, external preprocessors and ast plugin hooks *)
+(** Driver for the parser, external preprocessors and ast plugin hooks
+
+    Compiler-libs: beware this module makes no compatibility guarantees.
```

Comment:
Same suggestion as above. I'll leave off the repetition of the same thing from here.